backend/app/cmd/server.go

```diff
@@ -304,6 +304,9 @@ func (s *ServerCommand) HandleDeprecatedFlags() (result []DeprecatedFlag) {
 		s.SMTP.TimeOut = s.Auth.Email.TimeOut
 		result = append(result, DeprecatedFlag{Old: "auth.email.timeout", New: "smtp.timeout", RemoveVersion: "1.7.0"})
 	}
+	if s.Auth.Email.MsgTemplate != "email_confirmation.html.tmpl" {
+		result = append(result, DeprecatedFlag{Old: "auth.email.template", RemoveVersion: "1.9.0"})
```

Comment:
`HandleDeprecatedFlags` doesn't handle DeprecatedFlag without `New` option that well: it will be printed as "please use -- instead" anyway. We likely need some adjustment for that.